lib/ModelMixins/FeatureInfoUrlTemplateMixin.ts

```diff
@@ -104,7 +104,22 @@ function FeatureInfoUrlTemplateMixin<T extends AbstractConstructor<BaseType>>(
 
     wrapImageryPickFeatures<T extends ImageryProvider>(imageryProvider: T) {
       const realPickFeatures = imageryProvider.pickFeatures;
-      const catalogItem = this;
+
+      // Get CatalogItem properties, so we aren't reading them outside of a reaction
+      const featureInfoUrlTemplate = this.featureInfoUrlTemplate;
+      const proxiedFeatureInfoUrl = proxyCatalogItemUrl(
+        this,
+        this.featureInfoUrlTemplate ?? "",
```

Comment:
The statement on the line before this statement defines `featureInofUrlTemplate`. Since that is already defined, is there a reason to not use that here?